zyte_spider_templates_project/settings.py

```diff
@@ -9,25 +9,21 @@
     "zyte_spider_templates.spiders",
     "zyte_spider_templates_project.spiders",
 ]
+
 NEWSPIDER_MODULE = "zyte_spider_templates_project.spiders"
 
-CLOSESPIDER_TIMEOUT_NO_ITEM = 900
 ADDONS = {
     "scrapy_zyte_api.Addon": 500,
     "duplicate_url_discarder.Addon": 600,
+    "zyte_spider_templates.Addon": 700,
 }
-SCHEDULER_DISK_QUEUE = "scrapy.squeues.PickleFifoDiskQueue"
-SCHEDULER_MEMORY_QUEUE = "scrapy.squeues.FifoMemoryQueue"
+
 SPIDER_MIDDLEWARES = {
     "scrapy_poet.RetryMiddleware": 275,
-    "scrapy.spidermiddlewares.offsite.OffsiteMiddleware": None,
-    "zyte_spider_templates.middlewares.AllowOffsiteMiddleware": 500,
-    "zyte_spider_templates.middlewares.CrawlingLogsMiddleware": 1000,
 }
 
 # scrapy-poet
 SCRAPY_POET_DISCOVER = [
-    "zyte_spider_templates.pages",
```

Comment:
is it something which addon handles?

Reply:
[It should](https://github.com/zytedata/zyte-spider-templates-private/blob/805c9a8c186943ccd8032f0e6ba431b8e97107b0/zyte_spider_templates/_addon.py#L126-L128).

Reply:
Yes, it's there, but not sure if this aligned with this comment https://github.com/zytedata/zyte-spider-templates-project/pull/27#discussion_r1880403730

Reply:
Page objects should not impact spider listing, and at run time there is no issue, so this should be fine.